librz/include/rz_il/rzil_opcodes.h

```diff
@@ -118,6 +118,16 @@ struct rzil_op_cast_t {
 	RzILOp *val; ///< value to cast
 };
 
+
+/**
+ *  \struct rzil_op_concat_t
+ *  \brief op structure for concating bitv
```

Comment:
This does not add any value over the name the op already has, please describe exactly what it does, for example from BAP documentation:
> Appends x and y so that in the resulting vector the vector x occupies the most significant part and y the least significant part.

Reply:
Yes, append operates on two bitvectors, while concat operates on the list of bitvectors.